addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12509,7 +12509,11 @@ msgctxt "#24034"
 msgid "Check for updates"
 msgstr ""
 
-#empty string with id 24035
+#: Name of an Add-on category
+#: xbmc/addons/Addon.cpp
+msgctxt "#24035"
+msgid "Image collections"
```

Comment:
I'm fine with "image collections", but somehow "image packs" came to my mind when I thought about this feature.